.pipelines/azdo-base-pipeline.yml

```diff
@@ -1,7 +1,17 @@
 steps:
-- script: |
-    ./lint-and-test.sh
-  displayName: 'Linting & unit tests'
+- task: PythonScript@0
+  inputs:
+    scriptSource: inline
+    script: set -eux
+            flake8 --output-file=lint-testresults.xml --format junit-xml
+  displayName: 'Run lint tests'
+
+- task: PythonScript@0
+  inputs:
+    scriptSource: inline
+    script: set -eux 
```

Comment:
Haven't tested this yet, but my guess is you'll need to remove `set -eux` and add a `|` or move to a single line